docs/work_organization/ck.github_projects_process.reference.md

```diff
@@ -40,25 +40,28 @@
     style="width:6.5in;height:0.31944in" />
 
 - We use a base
-  [<u>template</u>](https://github.com/cryptokaizen/cmamp/projects?query=is%3Aopen+is%3Atemplate)
+  [<u>template</u>](https://github.com/orgs/causify-ai/projects?query=is%3Aopen+is%3Atemplate)
```

Comment:
Can we convert all these links into a markdown link?

We'll add a step to the linter at some point